inc/shortcodes/class-facebook.php

```diff
@@ -21,6 +21,11 @@ public static function get_shortcode_ui_args() {
 
 	public static function setup_actions() {
 		add_action( 'init', 'Shortcake_Bakery\Shortcodes\Facebook::action_init_register_scripts' );
+		add_action( 'shortcode_ui_after_do_shortcode', function( $shortcode ) {
+			if ( false !== stripos( $shortcode, '[facebook' ) ) {
```

Comment:
> How would you test output to TinyMCE?

I was thinking about calling https://github.com/fusioneng/Shortcake/blob/master/inc/class-shortcode-ui.php#L198 and inspecting the return value. To make it less of a hack, we could call a preview method.